### ISSUE TITLE
LDtk Haxe in-game API docs should mention how to draw

### ISSUE THREAD

**Issue by keir** [deepnight/ldtk#296]
The current LDtk Haxe API docs don't explain how to draw the level once imported. It would be great to offer some sample code for e.g. Heaps or gameBase.

I left another ticket on gameBase; unsure which repo this ticket should be on.
https://github.com/deepnight/gameBase/issues/25

Thanks for LDtk!

**Comment by deepnight** [comment]
You're right, I should definitely add some extra explanations. But it will be tied to Heaps.io engine, as I use only that one :)

**Comment by keir** [comment]
It could be a separate section below "Haxe API use"; perhaps "Using LDtk with Heaps.io".

Is it possible to contribute to the docs? I don't see them in the repo; only the JSON structure docs. Since I'm working through this (gamebase + LDtk integration) now, I could contribute some docs.

**Comment by deepnight** [comment]
The docs are only edited from the website directly (a wordpress). But I can see if I can grant some access if you want to 👍
